src/test/test-seccomp.c

```diff
@@ -323,6 +334,11 @@ static void test_protect_sysctl(void) {
 
                 assert_se(seccomp_protect_sysctl() >= 0);
 
+#if HAVE_VALGRIND_VALGRIND_H
+                if (!RUNNING_ON_VALGRIND)
```

Comment:
I don't understand this: why is the test reversed here?

Reply:
I assume this was a mistake. I force-pushed into your branch with this `if` reversed.

Reply:
Oh, thank you!